### PR TITLE
ffprobe, flac, id3tag, opusenc, vlc, wget, youtube-dl: add Italian translation

### PR DESCRIPTION
**Pull request by Agno94**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.it/common/ffprobe.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`ffprobe -v error -select_streams v:0 -show_entries stream=avg_frame_rate -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
+
+- Visualizza la larghezza o l'altezza di un video:
+
+`ffprobe -v error -select_streams v:0 -show_entries stream={{width|height}} -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
+
+- Visualizza il bit-rate medio di un video:
+
+`ffprobe -v error -select_streams v:0 -show_entries stream=bit_rate -of default=noprint_wrappers=1:nokey=1 {{input.mp4}}`
```
Comment (navarroaxel) [inline pages.it/common/ffprobe.md:24]: I `input` is not Italian. Please translate the arguments for get this easier for non-English speakers, you can use:
```suggestion
`ffprobe -v error -select_streams v:0 -show_entries stream=bit_rate -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
```
here

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.it/common/opusenc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# opusenc
+
+> Converte audio WAV o FLAC in Opus.
+> Maggiori informazioni: <https://opus-codec.org/docs/opus-tools/opusenc.html>.
+
+- Converte un file WAV in un file Opus usando le opzioni predefinite:
+
+`opusenc {{percorso/all/input.wav}} {{percorso/all/output}}.opus`
```
Comment (navarroaxel) [inline pages.it/common/opusenc.md:8]: You have `input` and `output` here.

File: pages.it/common/wget.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`wget {{https://esempio.com/abcd}}`
+
+- Scarica il contenuto dell'URL in un file (dal nome "efgh" in questo caso):
+
+`wget -O {{efgh}} {{https://esempio.com/abcd}}`
+
+- Scarica una singola pagina web e tutte le sue risorse(script, immagini, stili, ecc..) aspettando 3 secondi tra ogni richiesta:
+
+`wget --page-requisites --convert-links --wait=3 {{https://example.com/somepage.html}}`
```
Comment (navarroaxel) [inline pages.it/common/wget.md:17]: `somepage`?
```suggestion
`wget --page-requisites --convert-links --wait=3 {{https://esempio.com/somepage.html}}`
```

File: pages.it/common/youtube-dl.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`youtube-dl --format {{mp4}} -o "{{%(title)s di %(uploader)s del %(upload_date)s in %(playlist)s.%(ext)s}}" '{{url}}'`
+
+- Scarica, assieme al video, i sottotitoli in una lingua specificata :
+
+`youtube-dl --sub-lang {{it}} --write-sub '{{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}'`
+
+- Scarica una playlist, ne estrae l'audio e lo salva in formato mp3:
+
+`youtube-dl -i --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" '{{url della playlist}}'`
```
Comment (navarroaxel) [inline pages.it/common/youtube-dl.md:36]: The missing `snake_case` is a bug in the English page too:
```suggestion
`youtube-dl -i --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" '{{url_della_playlist}}'`
```
You can include that fix if you want.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.it/common/opusenc.md (hunk 2 above)
Comment (Agno94) [reply pages.it/common/opusenc.md:8]: I know. Still, you sometimes say `input` or `output` even in Italian in some case.
Some possible translations might be `file_di_input.wav`/`file_di_output.opus` or `file_originale.wav`/`file_modificato.opus`. I wonder which one is the best.
I see there is an Italian maintainer (@mebeim ), maybe he can give his opinion.

Comment (navarroaxel) [reply pages.it/common/opusenc.md:8]: Also @glenacota

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

/cc @glenacota for a 2nd opinion,, since they speak Italian.

Also, next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Comment by Agno94** [comment]
> Also, next time, perhaps you could avoid force-pushing please?

Yeah, I know it's bad. I just changed the last commit message adding `Co-authored-by: Axel Navarro <navarroaxel@gmail.com>` though.

File: pages.it/common/opusenc.md (hunk 2 above)
Comment (glenacota) [reply pages.it/common/opusenc.md:8]: I think input and output are pretty safe to use also in Italian. But I prefer your other proposal `file_originale.wav` and `file_modificato.opus`, though :)

File: pages.it/common/flac.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# flac
+
+> Codifica, decodifica e controlla file flac.
+> Maggiori informazioni: <https://xiph.org/flac>.
+
+- Converte un file wav in un file flac(questo creerà un file flac nella medesima posizione del file wav):
```
Comment (glenacota) [inline pages.it/common/flac.md:6]: Please add whitespace before the parenthesis

File: pages.it/common/flac.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`flac {{percorso/al/file.wav}}`
+
+- Codifica un file wav in flac, specificando il nome del risultato:
+
+`flac -o {{percorso/all/output.flac}} {{percorso/al/file.wav}}`
```
Comment (glenacota) [inline pages.it/common/flac.md:12]: For this and the next example, consider using your proposal "file_originale" and "file_modificato" (maybe, "file_convertito", in this case)

File: pages.it/common/id3tag.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# id3tag
+
+> Strumento per leggere, scrivere, e manipolare i tag(etichette) ID3v1 e ID3v2 di file MP3.
```
Comment (glenacota) [inline pages.it/common/id3tag.md:3]: Please add whitespace before the parenthesis

File: pages.it/common/vlc.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Riproduce un file:
+
+`vlc {{percorso/al/file}}`
+
+- Riproduce a schermo intero:
+
+`vlc --fullscreen {{percorso/al/file}}`
+
+- Riproduce mutato:
```
Comment (glenacota) [inline pages.it/common/vlc.md:14]: I don't think mutato is the right word here. Consider using "con audio disattivato" or "senza audio"

File: pages.it/common/vlc.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`vlc --no-audio {{percorso/al/file}}`
+
+- Riproduce ciclicamente:
```
Comment (glenacota) [inline pages.it/common/vlc.md:18]: Ciclicamente means periodically. I'd suggest using "Riproduce in ciclo continuo"

File: pages.it/common/wget.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+- Scarica il contenuto dell'URL in un file (dal nome "abcd" in questo caso):
+
+`wget {{https://esempio.com/abcd}}`
+
+- Scarica il contenuto dell'URL in un file (dal nome "efgh" in questo caso):
+
+`wget -O {{efgh}} {{https://esempio.com/abcd}}`
+
+- Scarica una singola pagina web e tutte le sue risorse(script, immagini, stili, ecc..) aspettando 3 secondi tra ogni richiesta:
```
Comment (glenacota) [inline pages.it/common/wget.md:15]: Please add whitespace before the parenthesis. Also, consider using "aspettando 3 secondi **dopo** ogni richiesta".

File: pages.it/common/wget.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+`wget --page-requisites --convert-links --wait=3 {{https://esempio.com/pagina_web.html}}`
+
+- Scarica tutti i file elencati nella cartella e nelle sue sotto-cartelle (non scarica gli elementi incorporati nella pagina)
```
Comment (glenacota) [inline pages.it/common/wget.md:19]: Please end the text with a colon

File: pages.it/common/youtube-dl.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+- Scarica l'audio di un video e lo converte in file MP3:
+
+`youtube-dl -x --audio-format {{mp3}} '{{url}}'`
+
+- Scarica l'audio di migliore qualità e il video di migliore qualità e li unisce:
+
+`youtube-dl -f bestvideo+bestaudio '{{url}}'`
+
+- Scarica video e li salva come file MP4 dal nomi personalizzati:
```
Comment (glenacota) [inline pages.it/common/youtube-dl.md:26]: Please fix plurals/singulars

File: pages.it/common/youtube-dl.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+`youtube-dl --format {{mp4}} -o "{{%(title)s di %(uploader)s del %(upload_date)s in %(playlist)s.%(ext)s}}" '{{url}}'`
+
+- Scarica, assieme al video, i sottotitoli in una lingua specificata :
```
Comment (glenacota) [inline pages.it/common/youtube-dl.md:30]: Please remove whitespace before the ending colon

**Review by glenacota — APPROVED** [review]
LGTM :)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**